### PR TITLE
Update enforce changelog

### PR DESCRIPTION
**Pull request by jamonation**
Enforce changelog update.

### CODE REVIEW

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/changelog.md [hunk 1]
```diff
@@ -2,14 +2,46 @@
...
+Release date: 2023-01-18
+#### Feature
+- Add support for AWS AppRunner services to cluster discovery.
+- Agentless discovery will now attempt to create an SQS queue subscribing to the audit log SNS topic in the region the cluster lives within.
+
+
+### v0.1.55
+Release date: 2023-01-17
+
+Customer facing changes: N/A
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/changelog.md:24]: Do we want this on the frontend site?

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/changelog.md [hunk 2]
```diff
@@ -2,14 +2,46 @@
...
-date: 2023-01-10 10:56:36 +0000 UTC
+date: 2023-01-18 18:11:40 +0000 UTC
 draft: false
 images: []
 weight: 799
 ---
 
 ## Introduction
-Any customer facing changes to Chainguard Enforce or [`chainctl`](/chainguard/chainguard-enforce/how-to-install-chainctl/) are highlighted in the following notes. Any new features, bug fixes, or general ease of use improvements will be listed under the corresponding release version.
+Any customer facing changes to Chainguard Enforce or [`chainctl`](/chainguard/chainguard-enforce/chainctl-docs/how-to-install-chainctl/) are highlighted in the following notes. Any new features, bug fixes, or general ease of use improvements will be listed under the corresponding release version.
```
Comment (jamonation) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/changelog.md:12]:
```suggestion
Any customer facing changes to Chainguard Enforce or [`chainctl`](/chainguard/chainguard-enforce/chainctl-docs/how-to-install-chainctl/) are highlighted in the following notes. Any new features, bug fixes, or general ease of use improvements will be listed under the corresponding release version.
```

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/changelog.md (hunk 1 above)
Comment (jamonation) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/changelog.md:24]: It's not ideal, but that's how it renders from the upstream release notes where there are no changes that impact customers. I think the future version that can merge content from academy and upstream could do away with this.

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/changelog.md (hunk 2 above)
Comment (jamonation) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/changelog.md:12]: The template upstream has the old link, so will need to update that until the introduction/hugo data can live here in Academy.

**Comment by jamonation** [comment]
@ltagliaferri could I get a +1 if the preview looks good to you? I had to force-reload my browser to get the latest 0.1.59 release showing here: https://deploy-preview-280--ornate-narwhal-088216.netlify.app/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/changelog/

**Review by ltagliaferri — APPROVED** [review]

**Pull request merged**